solr/core/src/java/org/apache/solr/util/circuitbreaker/CircuitBreakerManager.java

```diff
@@ -91,6 +92,16 @@ public void init(NamedList<?> args) {
     }
   }
 
+  @Override
+  public void close() throws IOException {
+    if (memEnabled) {
+      memCB.close();
+    }
+    if (cpuEnabled) {
+      cpuCB.close();
+    }
```

Comment:
edge case: `memCB.close()` throwing an exception and `cpuCB.close()` not getting called as a result. might not be worth extra complexity of handling since class is deprecated?

Reply:
Done